include/llvm/Support/raw_ostream.h

```diff
@@ -82,10 +85,13 @@ class raw_ostream {
     SAVEDCOLOR
   };
 
+
```

Comment:
Redundant empty lines? ;)

Reply:
Yeah. I was going to fix them if I had to fix something else. So it looks like I am.